src/cBiff12Writer.cls

```diff
@@ -235,7 +256,7 @@ EH:
     Resume Next
 End Function
 
-Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft As Long, ByVal lTop As Long, ByVal lWidth As Long, ByVal lHeight As Long)
+Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft As Long, ByVal lTOp As Long, ByVal lWidth As Long, ByVal lHeight As Long)
```

Comment:
Can we keep `lTop` param name here too?

Reply:
Changes not made by myself... Leave it unchanged

Reply:
Take the real changes in init, addshhet and in biff12part and leave Case sesitivity by yourself. Vb does no Checks for that